test/rewrite_quantization_test.cpp

```diff
@@ -82,7 +85,7 @@ TEST_CASE(quantizelinear)
     EXPECT(any_of(*p1.get_main_module(), &is_quantizelinear));
     EXPECT(none_of(*p2.get_main_module(), &is_quantizelinear));
     // ensure clip literals created in quantized program are scalar
-    EXPECT(any_of(*p2.get_main_module(), &is_clip_scalar));
+    EXPECT(migraphx::enabled(MIGRAPHX_ENABLE_CK{}) xor any_of(*p2.get_main_module(), &is_clip_scalar));
```

Comment:
if CK is enabled, then it should check if `clip` are not not scalar. This would just pass because it is doing `xor`ing